server/cmd/main.go

```diff
@@ -273,6 +276,32 @@ func main() {
 	connToInstanceTracker := handlers.ConnectionToStateMachineInstanceTracker{
 		ConnectToInstanceMap: make(map[uuid.UUID]*machines.StateMachine, 0),
 	}
+	
+		hosts, _, err := regManager.GetRegistrants(&v1alpha1.HostFilter{})
+		if err != nil {
+			log.Error(err)
+		}
+		for _, host := range hosts {
+			log.Infof("For Registrant %s Successfully imported %d models %d components %d relationships %d policy", host.Hostname, host.Summary.Models, host.Summary.Components, host.Summary.Relationships, host.Summary.Policies)
```

Comment:
@Jougan-0 are you using meshkit/logger? If not, please do.

Reply:
Yah lee we are using the logger from meshkit their were no function for adding arguments in between log statements do i went ahead and created 2 functions infof and errorf

Reply:
@nebula-aac I wonder what perspective you might have here.

Reply:
@leecalcote rather than creating the new fucntion and using it I used the combination of sprintf and log,info of the meshkit logger
Please may you have a look again